### ISSUE TITLE
Strange layout break when viewing pics

### ISSUE THREAD

**Issue by GoogleCodeExporter** [napsterrahul1/buddypress-media#17]
```
[What steps will reproduce the problem? (We need detailed, specific steps)]
1. Log in to BP
2. Go to members album
3. Open some pic

[What is the expected output? What do you see instead?]
When i am logged in and try to open some pic from member's album suddenly 
header, sidebar and footer disappear? On other pages i can see them normally.

When i am not logged in to BP this problem does not come up and layout is there 
even when i am opening single pic.

Any solution to this?

[What version of BP Media are you using?]


[What version of WordPress are you using?]


[What is the URL of your web host?]


[If you are running a public site, what is the URL to a page that shows the
problem?]


[Do you know how to use phpMyAdmin?]


[Do you know how to edit PHP code?]


[Please provide any additional information below]

```

Original issue reported on code.google.com by `tripatw...@gmail.com` on 23 Jul 2010 at 1:12

**Comment by GoogleCodeExporter** [comment]
```
We need to know what version of BP Media, WordPress, and BuddyPress you're 
using before we can try to reproduce this problem.
```

Original comment by `CarlRoett@gmail.com` on 23 Jul 2010 at 9:02

**Comment by GoogleCodeExporter** [comment]
```
Hi! I have this problem too.

BP Media: 1.0.7
BP: 1.2.5.2
WP: 3.0.1

see it here: http://bebetekep.hu/proba/members/admin/album/picture/2/
```

Original comment by `horvath....@gmail.com` on 3 Aug 2010 at 11:43

**Comment by GoogleCodeExporter** [comment]
```
Since it looks like you're just getting started with your site, please try 
installing version 0.1.9, available on the downloads page:

http://code.google.com/p/buddypress-media/downloads/list

...and report back here whether or not it works. 

DO NOT LEAVE 0.1.9 installed on a public site ...if you do, you'll lose all 
uploaded media items when we release the final version.

Thanks!

^F^
```

Original comment by `CarlRoett@gmail.com` on 4 Aug 2010 at 5:13

**Comment by GoogleCodeExporter** [comment]
```
Perhaps the same issue, but single image view:

[devsite]/members/admin/media/albums/1/item/1/

Breaks the sidebar - Info, licensing, and location come up below the activity 
stream for photo. Native sidebar goes away. Excessive whitespace below. Footer 
still present

As you can see by the link, its my first picture - but thought I would send 
them as the come.

WP: 3.0.1
BP: 1.2.5.2
Album+: 0.1.9 (9-14 nightly)
Theme: BP Default (with custom child theme)

Appears to be a missing </div> tag - somewhere before the <div 
class="bpa_right"> element. Though that only gets part of the issue - the 
details still get tagged awkwardly on the end.

```

Original comment by `pfstropa...@gmail.com` on 17 Sep 2010 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
User pfstropasia

This issue tracker entry is for version 0.1.7 of the plugin. 

Based on your description, you have found a NEW defect in version 0.1.9 of the 
plugin. Please create a NEW issue tracker entry for this defect. We will fix it 
as quickly as possible.

```

Original comment by `CarlRoett@gmail.com` on 17 Sep 2010 at 4:53
- Changed state: **Queued**

**Comment by GoogleCodeExporter** [comment]
Original comment by `CarlRoett@gmail.com` on 17 Sep 2010 at 9:37

**Comment by GoogleCodeExporter** [comment]
```
Apologies - just trying to minimize repeats. I'll repost.
```

Original comment by `pfstropa...@gmail.com` on 18 Sep 2010 at 2:58
